templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -371,6 +364,12 @@ export class {{ service.name }}Client {
           protosTypes{{ util.toInterface(method.outputInterface) }},
           protosTypes{{ util.toInterface(method.inputInterface) }}|undefined,
           {}|undefined>): void;
+/**
```

Comment:
I'm starting to think it makes perfect sense to have one `util` function combining these four lines :)

Reply:
Sure, that's a great idea!